datasets/Turkish_NER/Turkish_NER.py

```diff
@@ -0,0 +1,196 @@
+# coding=utf-8
```

Comment:
can you move your dataset from `Turkish_NER` to `turkish_ner` ?
We're using the snake case convention

Can you also rename the dataset script accordingly  ?